sample/index.php

```diff
@@ -4,12 +4,12 @@
 
 use Xwilarg\Discord\OAuth2;
 
-// Sample JSON configuration file, contains the following fields:
 // clientId: Client ID of the application
 // secret: Secret of the application
 // url: The redirect URL (URL called after the user is logged in, must be registered in https://discordapp.com/developers/applications/[YourAppId]/oauth)
-$auth = json_decode(file_get_contents('Credentials.json'), true);
-$oauth2 = new OAuth2($auth["clientId"], $auth["secret"], $auth["url"]);
+
```

Comment:
Could you also please remove this empty line and change the name in the comments to match the new argument names?

Reply:
Added.